### PR TITLE
Oceanwater 803 replace print with rospy method

### PR DESCRIPTION
**Pull request by AstroStucky**
## Linked Issues:
| EPIC ⚡ | N/A |
| :----------- | :----------- |
| Jira Ticket 🎟️ | [OCEANWATER-803](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-803) |
| Github :octocat:  | # |

## Summary of Changes
* Replaced all uses of print command with the appropriate rospy.log* function.
* All services in path_planning_commander.py now return using the same log and return function

## Review
rospy has several logging functions depending on the type of message. Please ensure I used the one that makes the most sense for the message.

## Test
I've already tested all arm services on my own, and since this PR mostly just replaces I/O functions, I don't think there is a need for reviewers to test every arm services. Just try out one of the following arm services: Grind, DigLinear, DigCircular, or Deliver. If it logs using rospy (you'll know because there's a timestamp in front of the message), and completes successfully, then your test passes.

### CODE REVIEW

**Comment by AstroStucky** [comment]
This overlaps, and is likely made redundant by a concurrent PR, #173. Lets hold off on reviews for now.

**Comment by Samahu** [comment]
> This overlaps, and is likely made redundant by a concurrent PR, #173. Lets hold off on reviews for now.

So we wait until #173 get's merged first before we review this?

**Comment by AstroStucky** [comment]
I wanted to finish my review of #173 before commenting on what to do with this PR. Since #173 was such a major refactor, I do not consider merging this PR worthwhile. Furthermore, since #173 was a refactor anyways, and they were minor, I did suggest some changes to #173 that satisfy OCEANWATER-803 by requesting that some print function calls be replaced with rospy.log* calls.

I think what makes the most sense is to just close this PR without merging.

**Comment by AstroStucky** [comment]
@Samahu Pointed out in our meeting this morning that this PR contains changes not entirely captured or made redundant in #173. Good catch! I will plan to merge this PR when #173 is merged, and handle merge conflicts.

**Comment by anjan1984** [comment]
I have merged my branch into melodic devel. You can merge melodic devel in your branch and it should resolve the conflicts.

**Comment by AstroStucky** [comment]
Thanks Anjan! melodic-devel is now merged in. You both may now start your reviews.

File: ow_lander/scripts/path_planning_commander.py
Comment (Samahu) [inline ow_lander/scripts/path_planning_commander.py:236]: There is inconsistency in what of some of the finish message printed across different activities. For example, some mention "Guarded Move activity completed" and some would be "Stow arm activity completed". Did you intentionally want this? To me logically you should have just done as follows and remove the word _arm_ when this method is invoked:
```suggestion
      rospy.loginfo("%s arm activity completed", activity_name)
      return success, "Done"
```

**Review by Samahu — CHANGES_REQUESTED** [review]
minor but valid comments

File: ow_lander/scripts/path_planning_commander.py
Comment (AstroStucky) [reply ow_lander/scripts/path_planning_commander.py:194]: Thanks, I think I meant to refactor all these, and missed one.

File: ow_lander/scripts/path_planning_commander.py
Comment (AstroStucky) [reply ow_lander/scripts/path_planning_commander.py:236]: I agree. This seems more consistent. Thanks.

**Review by Samahu — APPROVED** [review]

**Pull request merged**
